openshift/templates/metabase/metabase-postgres.dc.json

```diff
@@ -0,0 +1,268 @@
+{
+  "kind": "Template",
+  "apiVersion": "v1",
+  "metadata": {
+    "name": "metabase-postgres-dc"
```

Comment:
Isn't there a metabase-epic as well? do we use the same template for that DC? If not we should include that template in this folder or change this template name to just metabase-dc or something like that I think.

Reply:
yeah that's along the lines of my comment, 1 template with both in configs in it versus 2 separate files

Reply:
ah, I thought the whole metabase-dc file was getting deleted. I blame pre-coffee brain... Would be nice for these to be combined into one file though for easier deployment.  Would like to see a "role" label on the other template as well for the network security policies like this template does :)